src/hotspot/cpu/ppc/frame_ppc.inline.hpp

```diff
@@ -117,6 +117,8 @@ inline intptr_t* frame::link() const {
   return (intptr_t*)callers_abi()->callers_sp;
 }
 
+inline bool frame::can_access_link(Thread *thread) const { return true; }
```

Comment:
Why are ppc and s390 different from other platforms? If there is a valid reason, could you please add a short comment?

Reply:
Because they do not (as I see it) directly dereference a location on the stack to get to this value.